internal/docs/exported_fields.go

```diff
@@ -61,6 +63,9 @@ func renderFieldsTable(builder *strings.Builder, collected []fieldsTableRecord)
 	if metricTypesPresent {
 		builder.WriteString(" Metric Type |")
 	}
+	if valuesPresent {
+		builder.WriteString(" Value |")
+	}
```

Comment:
Making it optional makes most sense, as there's no change in the spec required for it to work. The same goes for **example** and other keys. It would be nice to see that as part of the documentation, next to the description key.

Reply:
Cycling back to this... @jsoriano do you have a suggestion on how and where to make this optional? I've been looking at the profiles, but want to make sure it's aligned with the vision you have about this.

Reply:
I see two options, each one with pros and cons, and can be complementary, so we could make both:

* Make this dependant on the `format_version` of the package. We would need to somehow pass this spec version to `renderExportedFields`, and maybe we could add a new option to `fields.InjectFieldsOptions`, to avoid loading values of constant keywords. The pro of this option is that all packages will eventually do this, the con, that packages will need to update its format version to have these fields.
* Add a build option (to `_dev/build/build.yml` for example) that enforces the generation of these fields. This would need changes in the spec, and would also need to add the setting in packages.

I would prefer the first option, but we have to take into account that this will only apply to packages using future versions of the spec.